pkg/generic/generic_service_test.go

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2021 CloudWeGo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package generic
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/cloudwego/kitex/internal/test"
+	gthrift "github.com/cloudwego/kitex/pkg/generic/thrift"
+	"github.com/cloudwego/kitex/pkg/remote"
+	codecThrift "github.com/cloudwego/kitex/pkg/remote/codec/thrift"
+	"github.com/cloudwego/kitex/pkg/serviceinfo"
+)
+
+type testImpl struct{}
+
+// GenericCall test call
+func (g *testImpl) GenericCall(ctx context.Context, method string, request interface{}) (response interface{}, err error) {
```

Comment:
Try to use mockgen as a way to construct mock structures, then you can do mock tests based on the generated structure, including checking the call times of a function or mocking a result to help testing.

Reply:
Mockgen is not yet enforced, but we'll be rolling out specs to make sure it's standard in the future

Reply:
you mean rename the structure to mock or something

Reply:
You can refer to [this document](https://github.com/golang/mock) to learn the usage of mockgen. You can put the files generated by `mockgen` in the `internal/mocks` directory.

Reply:
Is fixed now.